web/src/reactions.js

```diff
@@ -260,7 +266,6 @@ export function add_reaction(event) {
         });
 
         message.clean_reactions.set(local_id, clean_reaction_object);
-        const should_display_reactors = check_should_display_reactors(message.clean_reactions);
```

Comment:
Do I read correctly that this is the only change in this commit that actually runs different code -- doing this append calculation manually rather than just using the `clean_reactions` data structure?

I think the value you push onto `reaction_counts_and_user_ids` is missing the `count: 1` field, right, which happens to have no effect, but seems not intended?

Reply:
It does look like it's intended in that you declare the type to accept no `count` there and do the right thing. We can address this in a follow-up, but it seems like it might be cleaner if `count` were just always present?

Reply:
Ah yeah we can definitely set count to be 1 there, thanks.